internal/eval/eval.go

```diff
@@ -1634,7 +1634,13 @@ func evalCommandCount() []byte {
 	return clientio.Encode(diceCommandsCount, false)
 }
 
+// evalCommandGetKeys helps identify which arguments in a redis command
+// are interpreted as keys.
+// This is useful in analying long commands / scripts
 func evalCommandGetKeys(args []string) []byte {
+	if len(args) == 0 {
+		return diceerrors.NewErrWithMessage("wrong number of arguments for 'command getkeys'")
```

Comment:
Please use the error helper for this. We have a method called NewErrArity to handle these cases.

Reply:
Resolved. Thanks for the feedback.